openapiconverter/main.go

```diff
@@ -0,0 +1,189 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/CiscoDevNet/terraform-provider-meraki/gen"
+	"github.com/tidwall/gjson"
+	"gopkg.in/yaml.v2"
```

Comment:
If possible let's use "v3" as this is what we are already using in the generator code.

Reply:
changed to v3